Jenkinsfile

```diff
@@ -141,58 +219,217 @@ pipeline {
+                                        xunit([GoogleTest(deleteOutputFiles: true, failIfNotNew: true, pattern: "src/brave_unit_tests.xml", skipNoTestFiles: false, stopProcessingIfError: true)])
                                     }
                                     catch (ex) {
-                                        currentBuild.result = 'UNSTABLE'
+                                        currentBuild.result = "UNSTABLE"
                                     }
-                                    xunit([GoogleTest(deleteOutputFiles: true, failIfNotNew: true, pattern: 'src/brave_unit_tests.xml', skipNoTestFiles: false, stopProcessingIfError: true)])
                                 }
                             }
                         }
-                        stage('test-browser') {
+                        stage("test-browser") {
                             steps {
                                 script {
                                     try {
-                                        sh 'npm run test -- brave_browser_tests Release --output brave_browser_tests.xml'
-                                        xunit([GoogleTest(deleteOutputFiles: true, failIfNotNew: true, pattern: 'src/brave_browser_tests.xml', skipNoTestFiles: false, stopProcessingIfError: true)])
+                                        sh "npm run test -- brave_browser_tests ${BUILD_TYPE} --output brave_browser_tests.xml"
+                                        xunit([GoogleTest(deleteOutputFiles: true, failIfNotNew: true, pattern: "src/brave_browser_tests.xml", skipNoTestFiles: false, stopProcessingIfError: true)])
                                     }
                                     catch (ex) {
-                                        currentBuild.result = 'UNSTABLE'
+                                        currentBuild.result = "UNSTABLE"
                                     }
                                 }
                             }
                         }
-                        stage('dist') {
+                        stage("dist") {
+                            steps {
+                                sh "npm run create_dist -- ${BUILD_TYPE} --channel=${CHANNEL} --debug_build=false --official_build=true --skip_signing"
+                            }
+                        }
+                        stage("archive") {
+                            steps {
+                                // commented because it takes much longer to copy to Jenkins thant to S3
+                                // archiveArtifacts artifacts: "${OUT_DIR}/unsigned_dmg/*.dmg", fingerprint: true
+                                withAWS(credentials: "mac-build-s3-upload-artifacts", region: "us-west-2") {
+                                    awsIdentity()
+                                    s3Upload(acl: "Private", bucket: "${BRAVE_ARTIFACTS_BUCKET}", includePathPattern: "unsigned_dmg/*.dmg",
```

Comment:
Next step will be to upload the .pkg when release channel.

Reply:
@mbacchi is that captured in an issue yet?